neptune/projects.py

```diff
@@ -417,15 +416,47 @@ def _get_experiment_link(self, experiment):
         )
 
     def create_notebook(self):
-        notebook = self.client.create_notebook(self)
-        return notebook
+        """Create a new notebook in Neptune.
```

Comment:
@lukasz-walkiewicz here we need more explanation from data scientist point of view. What is the effect of this function. What sort of notebook is created - here we do not even pass ipynb file, so you need to explain it.